LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -1697,13 +1701,13 @@ private static void GetIteConfigurationsB(ISuperIO superIO, Manufacturer manufac
                         v.Add(new Voltage("VDDP", 5, 0, 1));
                         v.Add(new Voltage("DRAM", 6, 0, 1));
                         v.Add(new Voltage("+3V Standby", 7, 10, 10));
-                        v.Add(new Voltage("VBat", 8, 10, 10));
+                        v.Add(new Voltage("CMOS Battery", 8, 10, 10));
                         t.Add(new Temperature("System #1", 0));
-                        t.Add(new Temperature("Chipset", 1));
+                        t.Add(new Temperature("EC_TEMP1", 1));
```

Comment:
Is there a better name for this? TEMP1 seems duplicate as it's already a temperature.

Reply:
Since this is for the external temperature sensors you’d put on the motherboard, the name would likely be `External #` or something similar. Just saying External however is rather vague anyways

I’ve seen some ASUS and ASRock motherboards also identify these headers as `T_Sensor` (Temperature Sensor, basically duplicate name, yet still vague), and on some Dell motherboards as `Air Sensor` (I suppose this is somewhat of an acceptable name since this sensor would likely be placed somewhere in a case or outside to reflect ambient temperature).